pipeline-model-definition/src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/parser/ModelParser.groovy

```diff
@@ -186,6 +187,15 @@ class ModelParser implements Parser {
         return null // no 'pipeline', so this doesn't apply
     }
 
+    private @CheckForNull isFailFast(ModelASTOptions options){
+        if( options == null || options.options.isEmpty()) return false
+        def model = new ModelASTOption( new ModelASTMethodCall() )
+        model.name = "parallelsAlwaysFailFast"
+        model.args = []
+
+        return options.options.contains( model )
```

Comment:
This could probably be reworked to be something like `return options.options.any { it.name == "parallelsAlwaysFailFast" }` to avoid creating the `model`.